### PR TITLE
LegalEntityManagement endpoints

### PR DESCRIPTION
**Pull request by jillingk**
**Description**
Generated LEM endpoints. 

**Tested scenarios**
UT's

Please first approve (#690 and #668)

### CODE REVIEW

**Comment by AlexandrosMor** [comment]
The `httpclient` should be done in one pull request as it is here #690 and not in this one. This one should be only for `LegalEntityManagement endpoints`

File: Adyen/HttpClient/HttpClientWrapper.cs [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+using Adyen.HttpClient.Interfaces;
+using Adyen.Model;
+
+namespace Adyen.HttpClient
```
Comment (AlexandrosMor) [inline Adyen/HttpClient/HttpClientWrapper.cs:37]: The `httpclient` update should be done in one pull request as it is [here](https://github.com/Adyen/adyen-dotnet-api-library/pull/690) and not in this one. This one should be only for LegalEntityManagement endpoints

Comment (jillingk) [reply Adyen/HttpClient/HttpClientWrapper.cs:37]: Yep noted, however I did need the Restful client for these endpoints. Hence doing it this way and asking to approve the previous pr's (that include the client and automation script) first :)

**Review by michaelpaul — CHANGES_REQUESTED** [review]
Good job! Just one small fix to go.

File: Adyen/Constants/ClientConfig.cs [hunk 2]
```diff
@@ -56,6 +56,9 @@ public class ClientConfig
         public static string NexoProtocolVersion = "3.0";
         public static string BinLookupPalSuffix = "/pal/servlet/BinLookup/";
         public static string BinLookupApiVersion = "v50";
+        public static string LegalEntityManagementEndpointTest = "https://kyc-test.adyen.com/lem/";
+        public static string LegalEntityManagementEndpointLive = "https://kyc-test.adyen.com/lem/";
```
[the remark this answers is not in the document]
Comment (jillingk) [reply Adyen/Constants/ClientConfig.cs:60]: Wow good catch and stupid mistake from my end.

**Review by michaelpaul — APPROVED** [review]

**Review by AlexandrosMor — APPROVED** [review]

**Pull request merged**